src/components/tabs/Tabs.stories.mdx

```diff
@@ -16,3 +17,7 @@ import Tabs from './Tabs';
     </Tabs>
   </Story>
 </Preview>
+
+### Tab
```

Comment:
![](https://folk.ntnu.no/johannkv/img/EQf4oNT4W5.png)
Would it make sense to display a single tab here?
Or perhaps make the 
```md
### Tab
```
bigger, i.e `##` or `#`
I kinda feel some information is missing regarding what the subsection below the component is all about.

Reply:
We'll have to take a bigger pass on properties and documentation later, the point here is that Tabs doesn't have props, but Tab does